src/main/java/org/cryptomator/linux/quickaccess/DolphinPlaces.java

```diff
@@ -0,0 +1,172 @@
+package org.cryptomator.linux.quickaccess;
+
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.common.DisplayName;
+import org.cryptomator.integrations.common.OperatingSystem;
+import org.cryptomator.integrations.common.Priority;
+import org.cryptomator.integrations.quickaccess.QuickAccessService;
+import org.cryptomator.integrations.quickaccess.QuickAccessServiceException;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+/**
+ * Implemenation of the {@link QuickAccessService} for KDE desktop environments using Dolphin file browser.
+ */
+@DisplayName("KDE Dolphin Places")
+@OperatingSystem(OperatingSystem.Value.LINUX)
+@Priority(90)
+public class DolphinPlaces implements QuickAccessService {
+
+	private static final int MAX_FILE_SIZE = 1 << 15; //xml is quite verbose
+	private static final Path PLACES_FILE = Path.of(System.getProperty("user.home"), ".local/share/user-places.xbel");
+	private static final Path TMP_FILE = Path.of(System.getProperty("java.io.tmpdir"), "user-places.xbel.cryptomator.tmp");
+	private static final Lock MODIFY_LOCK = new ReentrantLock();
+	private static final String ENTRY_TEMPLATE = """
+			<bookmark href="%s">
+			 <title>%s</title>
+			 <info>
+			  <metadata owner="http://freedesktop.org">
+			   <bookmark:icon name="drive-harddisk-encrypted"/>
+			  </metadata>
+			  <metadata owner="https://cryptomator.org">
+			  	<id>%s</id>
+			  </metadata>
+			 </info>
+			</bookmark>""";
+
+
+	private static final Validator xmlValidator;
```

Comment:
constants should be named `XML_VALIDATOR`

Reply:
Done in https://github.com/cryptomator/integrations-linux/pull/85/commits/11d5bbdc7eb6e67da766e54730d841c2d1cab7ab